temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflowWithCronScheduleImpl.java

```diff
@@ -46,23 +48,20 @@ public String execute(String testName) {
       return null;
     }
 
-    lastCompletionResult = Workflow.getLastCompletionResult(String.class);
     lastFail = Workflow.getPreviousRunFailure();
+    retryCount.computeIfAbsent(testName, k -> new AtomicInteger());
```

Comment:
It should be
```
int count = retryCount.computeIfAbsent(testName, AtomicInteger::new).incrementAndGet();
    lastCompletionResults.computeIfAbsent(testName, HashMap::new).put(count, Workflow.getLastCompletionResult(String.class));
```
instead. Don't make lookups for something `computeIfAbsent` is already returning back.